lib/commands/log.js

```diff
@@ -29,8 +29,8 @@ extensions.startLogCapture = async function () {
   try {
     await this.logs.syslog.startCapture();
   } catch (err) {
-    log.warn(`Could not capture logs from the device: ${err.message}`);
-    log.debug('Continuing without capturing logs.');
+    log.warn(err.message);
```

Comment:
Is this error message self-explanatory? Why remove the preamble to give it context?

Reply:
changed